api-server/package.json

```diff
@@ -60,24 +42,20 @@
     "date-fns": "^1.29.0",
     "debug": "^2.2.0",
     "dedent": "~0.7.0",
-    "dotenv": "^4.0.0",
+    "dotenv": "^6.0.0",
     "express-flash": "~0.0.2",
     "express-session": "^1.12.1",
     "express-state": "^1.2.0",
     "express-validator": "^5.0.3",
     "fetchr": "^0.5.37",
-    "gatsby": "next",
-    "gatsby-plugin-manifest": "next",
-    "gatsby-plugin-react-helmet": "next",
-    "gatsby-plugin-sitemap": "next",
     "googleapis": "^16.1.0",
     "helmet": "^3.1.0",
     "helmet-csp": "^2.1.0",
     "jsonwebtoken": "^8.2.1",
     "lodash": "^4.17.5",
     "loopback": "^3.16.2",
     "loopback-boot": "^2.27.1",
-    "loopback-connector-mongodb": "^3.3.0",
+    "loopback-connector-mongodb": "3.3.0",
```

Comment:
Had to pin, setup breaks on latest version. I will raise an issue to investigate